src/script/lua_api/l_item.cpp

```diff
@@ -211,8 +225,6 @@ int LuaItemStack::l_to_table(lua_State *L)
 		lua_setfield(L, -2, "count");
 		lua_pushinteger(L, item.wear);
 		lua_setfield(L, -2, "wear");
-		lua_pushlstring(L, item.metadata.c_str(), item.metadata.size());
-		lua_setfield(L, -2, "metadata");
```

Comment:
You should add metadata to the table, but it is tricky as many mods will expect a string here. Maybe it should be a string if only `""` key is present?